chromium_src/components/search_engines/brave_template_url_prepopulate_data_unittest.cc

```diff
@@ -88,7 +90,7 @@ TEST_F(BraveTemplateURLPrepopulateDataTest, UniqueIDs) {
   };
 
   for (size_t i = 0; i < arraysize(kCountryIds); ++i) {
-    prefs_.SetInteger(prefs::kCountryIDAtInstall, kCountryIds[i]);
+    prefs_.SetInteger(kCountryIDAtInstall, kCountryIds[i]);
```

Comment:
I think this was moved out of prefs:: to country_codes:: (e.g.: https://cs.chromium.org/chromium/src/components/search_engines/template_url_prepopulate_data_unittest.cc?g=0&l=110&dlp=chromium&dlf=src/components/search_engines/template_url_prepopulate_data_unittest.cc&dlc=9cf3d7f024aec446ac1d9865c20950ab76c653d5&dlr=116&dlgp=components/search_engines/template_url_prepopulate_data_unittest.cc&dlgr=chromium/chromium/src&drp=chromium&drf=src/components/search_engines/template_url_prepopulate_data_unittest.cc&drc=68aafc3c9ac6c6345799375ad3e8bb31cb808175&drr=117&drgp=components/search_engines/template_url_prepopulate_data_unittest.cc&drgr=chromium/chromium/src )

Reply:
will change in a follow up.